backend/src/main/resources/application.yaml

```diff
@@ -30,20 +30,20 @@ simple-report-initialization:
     - name: Abbott IDNow
       manufacturer: Abbott
```

Comment:
@benwarfield-usds We need to make some updates to the devices in our systems. Thoughts on how to best do this?

Reply:
In the short term I would probably just shim it in the output layer--not super pretty but simple enough when there are only a half dozen values to worry about. In the longer term, we should have sysadmin-only API endpoints.